style.css

```diff
@@ -0,0 +1,188 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  background-color: #191765;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+#top-navigation {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: auto;
+  height: 48px;
+  margin-top: 60px;
+}
+
+.name {
+  color: #fff;
+  text-decoration: none;
+}
+
+.fa {
+  color: #fff;
+}
+
+.text {
+  color: #fff;
+  text-decoration: none;
+}
+
+.profile {
+  padding-left: 24px;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+#side-nav {
+  width: 33.333%;
+  display: flex;
+  justify-content: flex-end;
+  color: #fff;
+  padding-right: 40px;
+}
+
+.fa-bars {
+  font-size: 30px;
+  width: 33.33%;
+}
+
+.header h1 {
+  text-align: center;
+  font-weight: 700;
+  color: #7d73fc;
+  line-height: 72px;
+  padding-top: 50px;
+}
+
+.header h2 {
+  text-align: center;
+  font-weight: 500;
+  color: #edecff;
+  line-height: 44px;
+  text-size-adjust: 32px;
+  padding-top: 67;
+}
+
+.header p {
+  display: block;
+  text-align: center;
+  width: 100%;
+  font-size: 30px;
```

Comment:
- [ ] Optional

The font size according to the Figma design you are following is 20px, don't you think the font size you have chosen looks too big for a paragraph. Almost as big as your main heading. I strongly recommend you change it.